GeoStat-Mobile/GeoStat.Common/Services/AzureCloudService.cs

```
@@ -41,7 +41,14 @@ private async Task InitializeAsync()
 
         public async Task SyncOfflineCacheAsync()
         {
-            await InitializeAsync();
+            try
+            {
+                await InitializeAsync();
+            }
+            catch (Exception s)
+            {
```

empty catch statement won't lead to a good situations. At least, log it by IMvxLogger